### PR TITLE
Shellcheck Tested with Pass & Fixes.

### PR DESCRIPTION
**Pull request by Uniminin**
Commit contains fixes by shellcheck tests with shellcheck pass. Thank you.
You Might check shellcheck on this hyper link <https://www.shellcheck.net>

### CODE REVIEW

**Comment by Uniminin** [comment]
@helmuthdu ping!

**Comment by helmuthdu** [comment]
Sorry Bro, and thank you very much for this amazing job!

File: fifo [hunk 1]
```diff
@@ -322,22 +335,22 @@ format_partitions() {
...
     if [[ $OPTION == y ]]; then
       [[ -z $3 ]] && select_filesystem || filesystem=$3
-      mkfs.${filesystem} $1 \
-        $([[ ${filesystem} == xfs || ${filesystem} == btrfs || ${filesystem} == reiserfs ]] && echo "-f") \
-        $([[ ${filesystem} == vfat ]] && echo "-F32") \
-        $([[ $TRIM -eq 1 && ${filesystem} == ext4 ]] && echo "-E discard")
-      fsck $1
-      mkdir -p $2
-      mount -t ${filesystem} $1 $2
+      mkfs."${filesystem}" "$1" \
```
Comment (TheStrix) [inline fifo:331]: Seems to break the step of actually formatting the chosen root partition.. 

I got error: mkfs.ext4: invalid blocks `` on device `/dev/sda6`

Reverting last 2 merges fixes it for me.

Comment (helmuthdu) [reply fifo:331]: Screenshot 2020-06-25 at 15 25 30
I have fixed this, but I will do more tests to confirm that everything is working as expected.

**Pull request merged**